src/main/java/sky/TaskList.java

```diff
@@ -0,0 +1,95 @@
+package sky;
+
+import sky.task.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The TaskList class contains the task list e.g., it has operations to add/delete tasks in the list.
+ */
+public class TaskList {
+    private List<Task> taskList;
+
+    public TaskList() {
+        this.taskList = new ArrayList<>();
```

Comment:
I did so as I wanted to lessen the burden on the reader by abstracting a tad further about the kind of list being used. However, you may be right that it might be redundant. 🤔